extension/json/include/json.json

```diff
@@ -116,22 +116,27 @@
       },
       {
         "id": 109,
+        "name": "field_appearance_threshold",
```

Comment:
This is a backwards incompatible change - the backwards compatible way of adding a new field is to add it to the end (with a previously unused field id) and to add a `default` value that is inline with what you expect if the field has not been set by the user (i.e. when reading an older version).